Tests/Initialize-BitbucketServerAutomationTest.ps1

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 
 & (Join-Path -Path $PSScriptRoot -ChildPath '..\BitbucketServerAutomation\Import-BitbucketServerAutomation.ps1' -Resolve)
-& (Join-Path -Path $PSScriptRoot -ChildPath '..\Carbon\Import-Carbon.ps1' -Resolve)
+& (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\Carbon\*\Import-Carbon.ps1' -Resolve)

```

Comment:
Change this to `Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\Carbon') -Force` so we don't have to worry about what version(s) might be installed and we'll always get the latest.

Reply:
Curious, so when should we be calling `Import-Carbon.ps1` vs importing normally with the `Import-Module` cmdlet?